packages/account/src/configs.ts

```diff
@@ -1,4 +1,6 @@
-export const FUEL_NETWORK_URL =
+// #region FUEL_NETWORK_URL
+export const FUEL_NETWORK_URL: string =
   typeof process !== 'undefined'
     ? process?.env?.FUEL_NETWORK_URL || 'http://127.0.0.1:4000/graphql'
     : 'http://127.0.0.1:4000/graphql';
+// #endregion FUEL_NETWORK_URL
```

Comment:
Let's rethink the usage of this snippet. It's useful internally, but is it useful externally? 🤔 

Are the downsides of using `http://localhost:4000` directly?

Or the snippet could also define a variable or something.

Some points to think about.

```ts
const LOCAL_FUEL_NETWORK = "http://127.0.0.1:4000/graphql"
```

Reply:
A constant `LOCAL_FUEL_NETWORK_URL` I think is a good option.
```ts
const LOCAL_FUEL_NETWORK = "http://127.0.0.1:4000/graphql"
```

In the documentation maybe an extra explicit on the constant and value?
```ts
const NODE_URL = LOCAL_FUEL_NETWORK ?? "http://127.0.0.1:4000/graphql"
```

Reply:
I understand this constant is useful to us, but I'm not sure about its role for our users.

I find it hard to document a _constant_ that points to a default static value that is dependent on user env configs.

Reply:
d47ec8d527c29efc017ffa24a30d2ceeac896332

I removed the constant from the documentation, in favour of using `http://127.0.0.1:4000/graphql` within the snippet.